utilities/bash/control.sh

```diff
@@ -297,9 +297,11 @@ if ${lkinit} ; then
     if ${lkrenew} ; then
         echo " --> kinit -R beforehand:"
         kinit -R
+        aklog -c cern.ch -k CERN.CH
     else
         echo " --> kinit beforehand:"
         kinit
+        aklog -c cern.ch -k CERN.CH
```

Comment:
I would actually do something like:
   1. `kinit` (or `kinit -R`);
   1. check that `kinit` did not fail;
   1. `aklog`;
   1. check that `aklog` did not fail;
   1. `klist`